src/prommis/uky/uky_flowsheet.py

```diff
@@ -1020,103 +1607,431 @@ def initialize_system(m):
-            (0, "SO4"): 3924.07,
-            (0, "Sc"): 0.10,
-            (0, "Sm"): 0.30,
-            (0, "Y"): 0.39,
+            (0, "HSO4"): 1e-7,
+            (0, "Cl"): 1400,
+            (0, "La"): 160,
+            (0, "Nd"): 121,
+            (0, "Pr"): 30,
+            (0, "SO4"): 1e-7,
+            (0, "Sc"): 149.2,
+            (0, "Sm"): 13,
+            (0, "Y"): 18,
+        },
+    }
+    tear_guesses8 = {
+        "flow_vol": {0: 6},
+        "conc_mass_comp": {
+            (0, "Al"): 1e-7,
+            (0, "Ca"): 1e-7,
+            (0, "Ce"): 1e-7,
+            (0, "Dy"): 1e-7,
+            (0, "Fe"): 1e-7,
+            (0, "Gd"): 1e-7,
+            (0, "La"): 1e-7,
+            (0, "Nd"): 1e-7,
+            (0, "Pr"): 1e-7,
+            (0, "Sc"): 1e-7,
+            (0, "Sm"): 1e-7,
+            (0, "Y"): 1e-7,
+        },
+    }
+    tear_guesses9 = {
+        "flow_vol": {0: 6},
+        "conc_mass_comp": {
+            (0, "Al"): 1e-7,
+            (0, "Ca"): 1e-7,
+            (0, "Ce"): 1e-7,
+            (0, "Dy"): 1e-7,
+            (0, "Fe"): 1e-7,
+            (0, "Gd"): 1e-7,
+            (0, "La"): 1e-7,
+            (0, "Nd"): 1e-7,
+            (0, "Pr"): 1e-7,
+            (0, "Sc"): 1e-7,
+            (0, "Sm"): 1e-7,
+            (0, "Y"): 1e-7,
         },
     }
 
     # Pass the tear_guess to the SD tool
     seq.set_guesses_for(m.fs.precipitator.cv_aqueous.properties_out[0], tear_guesses1)
     seq.set_guesses_for(m.fs.precipitator.cv_precipitate[0], tear_guesses2)
     seq.set_guesses_for(m.fs.leach.liquid_inlet, tear_guesses3)
-    seq.set_guesses_for(m.fs.solex_rougher.mscontactor.aqueous_inlet, tear_guesses4)
+    seq.set_guesses_for(
+        m.fs.solex_rougher_load.mscontactor.organic_inlet, tear_guesses4
+    )
+    seq.set_guesses_for(
+        m.fs.solex_rougher_load.mscontactor.aqueous_inlet, tear_guesses5
+    )
+    seq.set_guesses_for(
+        m.fs.solex_cleaner_load.mscontactor.organic_inlet, tear_guesses6
+    )
+    seq.set_guesses_for(
+        m.fs.solex_cleaner_load.mscontactor.aqueous_inlet, tear_guesses7
+    )
+    seq.set_guesses_for(m.fs.precip_sx_mixer.outlet, tear_guesses7)
+    seq.set_guesses_for(m.fs.rougher_org_make_up.outlet, tear_guesses8)
+    seq.set_guesses_for(m.fs.cleaner_org_make_up.outlet, tear_guesses9)
 
     def function(stream):
         initializer_feed = FeedInitializer()
         initializer_product = ProductInitializer()
-        initializer1 = MSContactorInitializer()
-        initializer2 = BlockTriangularizationInitializer()
+        initializer_sep = SeparatorInitializer()
+        initializer_mix = MixerInitializer()
+        initializer_leach = LeachingTrainInitializer()
+        initializer_bt = BlockTriangularizationInitializer()
 
         propagate_state(m.fs.liq_feed)
         propagate_state(m.fs.sol_feed)
+        propagate_state(m.fs.org_feed)
+        propagate_state(m.fs.org_feed2)
 
         if stream == m.fs.leach_liquid_feed:
```

Comment:
Why is this called "stream"? From the context, it appears that this is initializing unit operations thus the name is confusing me.

Reply:
Changed to "unit"